submit-llama3-1b.sh

```diff
@@ -235,6 +243,9 @@ if [ "$NSYS_PROFILER" = true ]; then
     TRAINING_CMD="$NSYS_LAUNCHER $TRAINING_CMD --profile"
 fi
 
+# Save sbatch script
+cat $0 >> $EXP_DIR/submit_run.sbatch
```

Comment:
```suggestion
cp $0 $DEBUG_DIR
```
`DEBUG_DIR` is defined as `DEBUG_DIR=$EXP_DIR/debug/$SLURM_JOB_ID`. Does it makes more sense to store the sbatch script for every job rather than to store the script for every experiment in case we change some parameters across runs which are not modifying the `EXP_NAME`?